server/internal/pki/pki.go

```diff
@@ -220,7 +220,32 @@ func (p *pki) worker() {
 			}
 		}
 
```

Comment:
it's not ideal to make this worker method even longer

Reply:
refactored into a method